lib/check_params.h

```diff
@@ -357,7 +357,11 @@ void printQudaInvertParam(QudaInvertParam *param) {
 #if defined INIT_PARAM
   P(gcrNkrylov, INVALID_INT);
 #else
-  if (param->inv_type == QUDA_GCR_INVERTER || param->inv_type == QUDA_CA_GCR_INVERTER) {
+  if (param->inv_type == QUDA_GCR_INVERTER ||
+      param->inv_type == QUDA_CA_GCR_INVERTER ||
+      param->inv_type == QUDA_CA_CG_INVERTER ||
+      param->inv_type == QUDA_CA_CGNE_INVERTER ||
+      param->inv_type == QUDA_CA_CGNR_INVERTER) {
```

Comment:
clang-format suggested the following change:


```
-  if (param->inv_type == QUDA_GCR_INVERTER ||
-      param->inv_type == QUDA_CA_GCR_INVERTER ||
-      param->inv_type == QUDA_CA_CG_INVERTER ||
-      param->inv_type == QUDA_CA_CGNE_INVERTER ||
-      param->inv_type == QUDA_CA_CGNR_INVERTER) {
+  if (param->inv_type == QUDA_GCR_INVERTER || param->inv_type == QUDA_CA_GCR_INVERTER
+      || param->inv_type == QUDA_CA_CG_INVERTER || param->inv_type == QUDA_CA_CGNE_INVERTER
+      || param->inv_type == QUDA_CA_CGNR_INVERTER) {
```

<!-- identifier: clang-format -->
<!-- File: lib/check_params.h -->